app/controllers/parks_controller.rb

```diff
@@ -23,6 +23,23 @@ def create
     end
   end
 
+  def edit
+    @park = Park.find(params[:id])
+  end
+
+  def update
+    @park = Park.find(params[:id])
+    @park.update(park_params)
+    if @park.save
+      flash[:notice] = "You have successfully updated your question!"
+      redirect_to @park
+    else
+      flash.now[:notice] = "Invalid input."
+      render 'edit'
+    end
+  end
+
+private
```

Comment:
Indent access modifiers like `private`.<br>Keep a blank line before and after `private`.